cmake/std/atdm/shiller/environment.sh

```diff
@@ -86,7 +82,15 @@ elif [[ "$ATDM_CONFIG_COMPILER" == "CUDA"* ]]; then
   # Avoids timeouts due to not running on seprate GPUs (see #2446)
 else
   echo "***"
-  echo "*** ERROR: COMPILER=$ATDM_CONFIG_COMPILER is not supported on this system!"
+  echo "*** ERROR: KOKKOS_ARCH=$ATDM_CONFIG_KOKKOS_ARCH with compiler=$ATDM_CONFIG_COMPILER "
```

Comment:
Another excellent error message!  These types of error messages will avoid a lot of questions from users.  Printing the set of valid KOKKOS_ARCH values for a given system and compiler is super valuable!